model.py

```diff
@@ -0,0 +1,281 @@
+        "ExpandVariableOperator",
+        "Number of times ExpandVariableOperator has been called",
+    ),
+    "FilterOperator": Gauge(
+        "FilterOperator", "Number of times FilterOperator has been called"
+    ),
+    "ForeachOperator": Gauge(
+        "ForeachOperator", "Number of times ForeachOperator has been called"
+    ),
+    "LimitOperator": Gauge(
+        "LimitOperator", "Number of times LimitOperator has been called"
+    ),
+    "MergeOperator": Gauge(
+        "MergeOperator", "Number of times MergeOperator has been called"
+    ),
+    "OnceOperator": Gauge(
+        "OnceOperator", "Number of times OnceOperator has been called"
+    ),
+    "OptionalOperator": Gauge(
+        "OptionalOperator", "Number of times OptionalOperator has been called"
+    ),
+    "OrderByOperator": Gauge(
+        "OrderByOperator", "Number of times OrderByOperator has been called"
+    ),
+    "ProduceOperator": Gauge(
+        "ProduceOperator", "Number of times ProduceOperator has been called"
+    ),
+    "RemoveLabelsOperator": Gauge(
+        "RemoveLabelsOperator", "Number of times RemoveLabelsOperator has been called"
+    ),
+    "RemovePropertyOperator": Gauge(
+        "RemovePropertyOperator",
+        "Number of times RemovePropertyOperator has been called",
+    ),
+    "ScanAllByIdOperator": Gauge(
+        "ScanAllByIdOperator", "Number of times ScanAllByIdOperator has been called"
+    ),
+    "ScanAllByLabelOperator": Gauge(
+        "ScanAllByLabelOperator",
+        "Number of times ScanAllByLabelOperator has been called",
+    ),
+    "ScanAllByLabelPropertyOperator": Gauge(
+        "ScanAllByLabelPropertyOperator",
+        "Number of times ScanAllByLabelPropertyOperator has been called",
+    ),
+    "ScanAllByLabelPropertyRangeOperator": Gauge(
+        "ScanAllByLabelPropertyRangeOperator",
+        "Number of times ScanAllByLabelPropertyRangeOperator has been called",
+    ),
+    "ScanAllByLabelPropertyValueOperator": Gauge(
+        "ScanAllByLabelPropertyValueOperator",
+        "Number of times ScanAllByLabelPropertyValueOperator has been called",
+    ),
+    "ScanAllOperator": Gauge(
+        "ScanAllOperator", "Number of times ScanAllOperator has been called"
+    ),
+    "SetLabelsOperator": Gauge(
+        "SetLabelsOperator", "Number of times SetLabelsOperator has been called"
+    ),
+    "SetPropertiesOperator": Gauge(
+        "SetPropertiesOperator", "Number of times SetPropertiesOperator has been called"
+    ),
+    "SetPropertyOperator": Gauge(
+        "SetPropertyOperator", "Number of times SetPropertyOperator has been called"
+    ),
+    "SkipOperator": Gauge(
+        "SkipOperator", "Number of times SkipOperator has been called"
+    ),
+    "UnionOperator": Gauge(
+        "UnionOperator", "Number of times UnionOperator has been called"
+    ),
+    "UnwindOperator": Gauge(
+        "UnwindOperator", "Number of times UnwindOperator has been called"
+    ),
+}
```

Comment:
```suggestion
operator_names = [
    "AccumulateOperator",
    "AggregateOperator",
    "ApplyOperator",
    "CallProcedureOperator",
    "CartesianOperator",
    "ConstructNamedPathOperator",
    "CreateExpandOperator",
    "CreateNodeOperator",
    "DeleteOperator",
    "DistinctOperator",
    "EdgeUniquenessFilterOperator",
    "EmptyResultOperator",
    "EvaluatePatternFilterOperator",
    "ExpandOperator",
    "ExpandVariableOperator",
    "FilterOperator",
    "ForeachOperator",
    "LimitOperator",
    "MergeOperator",
    "OnceOperator",
    "OptionalOperator",
    "OrderByOperator",
    "ProduceOperator",
    "RemoveLabelsOperator",
    "RemovePropertyOperator",
    "ScanAllByIdOperator",
    "ScanAllByLabelOperator",
    "ScanAllByLabelPropertyOperator",
    "ScanAllByLabelPropertyRangeOperator",
    "ScanAllByLabelPropertyValueOperator",
    "ScanAllOperator",
    "SetLabelsOperator",
    "SetPropertiesOperator",
    "SetPropertyOperator",
    "SkipOperator",
    "UnionOperator",
    "UnwindOperator",
]

INDEX = {
    name: Gauge(name, f"Number of times {name} has been called")
    for name in operator_names
}
```

Reply:
Maybe you can apply same for others. From readability perspective and DRY concept it makes sense to me.